tools/faker_docs_utils/mkdocs_plugins/main_mkdocs_plugin.py

```diff
@@ -58,9 +58,9 @@ def new_warning(self, *args, **kwargs):
             build_locales_env = os.environ.get(
                 "SF_MKDOCS_BUILD_LOCALES"
             ) or self.config.get("build_locales", None)
-            if build_locales_env == "False":
+            if build_locales_env in (False, "False", "false"):
                 locales_list = ["en_US", "fr_FR"]
-            elif build_locales_env in (True, "True", None):
+            elif build_locales_env in (True, "True", "true", None):
                 locales_list = None  # means "all"
             elif isinstance(build_locales_env, str):
                 locales_list = build_locales_env.split(",")
```

Comment:
I know this isn't the point of the PR, but wouldn't assigning the value directly to `locales_list` simplify things a bit? Meaning set `build_locales` to `None` by default and just use that. To override, set `SF_MKDOCS_BUILD_LOCALES: "en_US,fr_FR"`.